### PR TITLE
Update node verion to 14.15.0 LTS for MacOS and Windows

### PR DESCRIPTION
**Pull request by core1024**
Progress on https://github.com/Stremio/stremio-shell/issues/143

In order the build to pass we need new version of server.js compatible with node 14.15.0

### CODE REVIEW

**Review by TheSparrowLenz — COMMENTED** [review]
This build will fail due to the change to nodejs_version **14.15.0.**
server.js is still using `os.tmpDir()` which has been deprecated in this new nodejs versions.

A change in server.js from `os.tmpDir()` to `os.tmpdir()` should be enough for this to work.

**Comment by TheSparrowLenz** [comment]
> Progress on #143
> 
> In order the build to pass we need new version of server.js compatible with node 14.15.0


This build will fail due to the change to nodejs_version **14.15.0.**
server.js is still using one instance of `os.tmpDir()` which has been deprecated.

`os.tmpdir()` appears 4 times in server.js => 3 times correctly as `os.tmpdir()` and one time as `os.tmpDir()` on line **132531**.
Changing line **132531** will fix this.


This build will fail due to the change to nodejs_version **14.15.0.**
server.js is still using one instance of `os.tmpDir()` which has been deprecated.

`os.tmpdir()` appears 4 times in server.js => 3 times correctly as `os.tmpdir()` and one time as `os.tmpDir()` on line **132531**.
Changing line **132531** will fix this.

File: stremio.pro [hunk 1]
```diff
@@ -1,6 +1,6 @@
 TEMPLATE = app
 
-VERSION=4.4.120
+VERSION=4.4.122
```
Comment (Ivshti) [inline stremio.pro:3]: is this supposed to be part of it?

**Review by Ivshti — CHANGES_REQUESTED** [review]

File: stremio.pro (hunk 1 above)
Comment (core1024) [reply stremio.pro:3]: Perhaps I was testing. I'll remove this part

**Review by Ivshti — APPROVED** [review]

**Pull request merged**